lib/vehicle.js

```diff
@@ -139,6 +139,71 @@ Vehicle.prototype.permissions = async function(paging = {}) {
   return response;
 };
 
+/**
+ * @typedef ChargeLimit
+ * @property {number} limit - the charge limit expressed as a decimal value between 0 and 1.
+ */
+
+/**
+ * Fetch the charge limit for an electric vehicle
+ *
+ * @method
+ * @returns {ChargeLimit}
+ * @throws {SmartcarError} - an instance of SmartcarError.
+ *   See the [errors section](https://github.com/smartcar/node-sdk/tree/master/doc#errors)
+ *   for all possible errors.
+ *
+ * @example
+ * {
+ *   limit: .7,
+ *   meta: {
+ *    requestId: '26c14915-0c26-43c5-8e42-9edfc2a66a0f',
+ *   }
+ * }
+ */
+
+Vehicle.prototype.getChargeLimit = async function() {
+  const response = await this.service.request(
+    'get',
+    'charge/limit',
+  );
+
+  return response;
+};
+
+/**
+ * Set the charge limit for an electric vehicle.
+ *
+ * @method
+ * @param {number} chargeLimit - a number between 0 and 1
+ * @returns {ChargeLimit}
+ * @throws {SmartcarError} - an instance of SmartcarError.
+ *   See the [errors section](https://github.com/smartcar/node-sdk/tree/master/doc#errors)
+ *   for all possible errors.
+ *
+ * @example
+ * {
+ *   status: string,
+ *   meta: {
+ *    requestId: '26c14915-0c26-43c5-8e42-9edfc2a66a0f',
+ *   }
+ * }
+ */
+Vehicle.prototype.setChargeLimit = async function(chargeLimit) {
+
+  const body = {
+    limit: String(chargeLimit),
+  };
+
+  const response = await this.service.request(
+    'post',
+    'charge/limit',
+    body,
```

Comment:
```suggestion
    {body},
```

:smiling_face_with_tear:

Reply:
sadge